src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -159,6 +163,82 @@ public static ZoomLink parseZoom(String zoomLink) throws ParseException {
         return new ZoomLink(zoomLink);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code Name}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static seedu.address.model.event.Name parseEventName(String name) throws ParseException {
```

Comment:
Just made an observation that Name over here is duplicated for events. Just a note, I don't expect you to change this for now.